### PR TITLE
Rename some model types for Queue service

### PR DESCRIPTION
**Pull request by Jinming-Hu**
closes https://github.com/Azure/azure-sdk-for-cpp/issues/3026

# Pull Request Checklist

Please leverage this checklist as a reminder to address commonly occurring feedback when submitting a pull request to make sure your PR can be reviewed quickly:

See the detailed list in the [contributing guide](https://github.com/Azure/azure-sdk-for-cpp/blob/master/CONTRIBUTING.md#pull-requests).

- [x] [C++ Guidelines](https://azure.github.io/azure-sdk/cpp_introduction.html)
- [x] Doxygen docs
- [x] Unit tests
- [x] No unwanted commits/changes
- [x] Descriptive title/description
  - [x] PR is single purpose
  - [x] Related issue listed
- [x] Comments in source
- [x] No typos
- [x] Update changelog
- [x] Not work-in-progress
- [x] External references or docs updated
- [x] Self review of PR done
- [x] Any breaking changes?

### CODE REVIEW

**Review by vhvb1989 — APPROVED** [review]

File: sdk/storage/azure-storage-queues/CHANGELOG.md [hunk 1]
```diff
@@ -6,6 +6,14 @@
 
 ### Breaking Changes
 
+- Renamed `GetServicePropertiesResult` to `QueueServiceProperties`.
+- Renamed `GetServiceStatisticsResult` to `ServiceStatistics`.
+- Renamed `GetQueuePropertiesResult` to `QueueProperties`.
+- Renamed `GetQueueAccessPolicyResult` to `QueueAccessPolicy`.
+- Wrapped the first parameter of `QueueClient::SetAccessPolicy`, a vector of signed identifiers into a struct `QueueAccessPolicy`.
+- Renamed `ReceiveMessagesResult` to `ReceivedMessages`.
+- Renamed `PeekMessagesResult` to `PeekMessagesResult`.
```
Comment (ahsonkhan) [inline sdk/storage/azure-storage-queues/CHANGELOG.md:15]:
```suggestion
- Renamed `PeekMessagesResult` to `PeekedMessages`.
```

**Comment by Jinming-Hu** [comment]
Storage livetest doesn't work anymore. Do you guys know what's changed recently? @vhvb1989 @ahsonkhan @antkmsft @danieljurek

**Comment by danieljurek** [comment]
Running live storage tests against the `main` branch succeeds -- https://dev.azure.com/azure-sdk/internal/_build/results?buildId=1179177&view=logs&j=0e119b68-c922-5ad7-2413-fa5b213ae9b8&t=6a43d8b6-398a-5e96-c7bb-4c716253f214

I re-ran the pipeline with the failed live storage tests and some of those test runs are still failing. 

I did take out a `-V` from our testing command which prevents test information from displaying. That was a mistake. The same test logging information can also now be found in the `Tests` tab: 
image

You can drill into failed tests to see more information: 
image

I've got a PR to readd `-V` to our test run so the console output is more useful https://github.com/Azure/azure-sdk-for-cpp/pull/3042/files. Sorry for the confusion.

**Pull request merged**